docs/windows.md

```diff
@@ -234,8 +234,8 @@ spec:
       - command:
         - pwsh
         args:
-        - -file
-        - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/kube-proxy-windows/run-script.ps1
+          - -file
```

Comment:
this change leads to inconsistent indention between command's and args'

Reply:
updated